src/menus/EditMenus.cpp

```diff
@@ -1,36 +1,37 @@
 
 #include "../AdornedRulerPanel.h"
+#include "../AudioPasteDialog.h"
 #include "../Clipboard.h"
 #include "../CommonCommandFlags.h"
 #include "../LabelTrack.h"
 #include "../Menus.h"
 #include "../NoteTrack.h"
-#include "Project.h"
-#include "ProjectHistory.h"
-#include "ProjectRate.h"
 #include "../ProjectWindow.h"
 #include "../ProjectWindows.h"
 #include "../SelectUtilities.h"
-#include "SyncLock.h"
 #include "../TrackPanel.h"
 #include "../TrackPanelAx.h"
-#include "UndoManager.h"
-#include "ViewInfo.h"
-#include "WaveTrack.h"
-#include "WaveClip.h"
-#include "SampleBlock.h"
 #include "../commands/CommandContext.h"
 #include "../commands/CommandManager.h"
-#include "TimeWarper.h"
 #include "../prefs/PrefsDialog.h"
 #include "../prefs/TracksBehaviorsPrefs.h"
 #include "../tracks/labeltrack/ui/LabelTrackView.h"
 #include "../tracks/playabletrack/wavetrack/ui/WaveChannelView.h"
-#include "AudacityMessageBox.h"
 #include "../widgets/VetoDialogHook.h"
-#include "../AudioPasteDialog.h"
+#include "AudacityMessageBox.h"
 #include "BasicUI.h"
+#include "Project.h"
+#include "ProjectHistory.h"
+#include "ProjectRate.h"
+#include "ProjectTimeSignature.h"
+#include "SampleBlock.h"
 #include "Sequence.h"
+#include "SyncLock.h"
+#include "TimeWarper.h"
+#include "UndoManager.h"
+#include "ViewInfo.h"
+#include "WaveClip.h"
+#include "WaveTrack.h"
```

Comment:
I would prefer that you don't permute these lines.  Leave them alphabetical by the last path component.  As more files move to libraries, and the `#include` lines change, there is just removal of `../` and can make a smaller diff.

Reply:
Sure.